fearless/Modules/Staking/StakingUnbondConfirm/View/StakingUnbondConfirmLayout.swift

```diff
@@ -136,25 +157,34 @@ final class StakingUnbondConfirmLayout: UIView {
             make.leading.trailing.equalToSuperview().inset(UIConstants.horizontalInset)
         }
 
+        stackView.addArrangedSubview(collatorView)
+        collatorView.snp.makeConstraints { make in
+            make.width.equalTo(stackView)
+            make.height.equalTo(UIConstants.actionHeight)
+        }
+        stackView.setCustomSpacing(UIConstants.bigOffset, after: collatorView)
+
         stackView.addArrangedSubview(accountView)
         accountView.snp.makeConstraints { make in
             make.width.equalTo(stackView)
-            make.height.equalTo(52)
+            make.height.equalTo(UIConstants.actionHeight)
         }
 
-        stackView.setCustomSpacing(16.0, after: accountView)
+        stackView.setCustomSpacing(UIConstants.bigOffset, after: accountView)
         stackView.addArrangedSubview(amountView)
         amountView.snp.makeConstraints { make in
             make.width.equalTo(stackView)
-            make.height.equalTo(72.0)
+            make.height.equalTo(UIConstants.amountViewHeight)
         }
 
-        stackView.setCustomSpacing(16.0, after: amountView)
+        stackView.setCustomSpacing(UIConstants.bigOffset, after: amountView)
 
-        addSubview(networkFeeConfirmView)
+        addSubview(networkFeeFooterView)
 
-        networkFeeConfirmView.snp.makeConstraints { make in
+        networkFeeFooterView.snp.makeConstraints { make in
             make.leading.bottom.trailing.equalToSuperview()
         }
+
+        collatorView.isHidden = true
```

Comment:
this func for setup layout. move this code please to variable constructor (computed property initialization block)